kokoro/linux/dockerfile/test/php80/Dockerfile

```diff
@@ -90,6 +90,34 @@ RUN wget -O phpunit https://phar.phpunit.de/phpunit-9.phar \
   && cp phpunit /usr/local/php-8.0/bin \
   && mv phpunit /usr/local/php-8.0-zts/bin
 
+# php 8.1
```

Comment:
I wasn't sure whether the idea was to have one dockerfile per php version moving forward, or an all-in-one like is done for php7.x